app/util/AtomElementBuilders.scala

```diff
@@ -44,14 +44,29 @@ object AtomElementBuilders {
       title = createAtomFields.flatMap(_.title),
       id = java.util.UUID.randomUUID.toString,
       atomType = atomType,
-      defaultHtml = createAtomFields.flatMap(_.defaultHtml).getOrElse(buildDefaultHtml(atomType = atomType, atomData = defaultAtoms(atomType))),
+      defaultHtml = createAtomFields.flatMap(_.defaultHtml).getOrElse(buildHtml(atomType = atomType, atomData = defaultAtoms(atomType))),
       data = defaultAtoms(atomType),
       contentChangeDetails = buildContentChangeDetails(user, None, updateCreated = true)
     )
   }
 
   // this is just a stub - will eventually need to generate default HTML for all the atom types we support
-  def buildDefaultHtml(atomType: AtomType, atomData: AtomData): String = {
+  def buildHtml(atomType: AtomType, atomData: AtomData): String = atomType match {
+    case AtomType.Storyquestions => buildStoryQuestionsHtml(atomData)
+    case _ => buildDefaultHtml(atomType)
+  }
+
+  def buildStoryQuestionsHtml(sqData: AtomData): String = {
+    val questions: Option[String] = for {
+      eqs <- sqData.asInstanceOf[AtomData.Storyquestions].storyquestions.editorialQuestions
+    } yield {
+      eqs flatMap (_.questions map { q => s"<li>${q.questionText}</li>" }) mkString ""
+    }
+
+    questions map ("<ul>" ++ _ ++ "</ul>") getOrElse ""
```

Comment:
Out of curiosity: I understand you want to construct a list with the questions, but you seem to not wrap it in a `div` and I was wondering why.
I mean having something like:
```
<div class="atom-${atomType.name}"><ul>..</ul>...</div>
```